packages/angular/cli/lib/config/schema.json

```diff
@@ -847,7 +847,7 @@
             },
             "statsJson": {
               "type": "boolean",
-              "description": "Generates a 'stats.json' file which can be analyzed using tools such as: #webpack-bundle-analyzer' or https://webpack.github.io/analyse .",
+              "description": "Generates a 'stats.json' file which can be analyzed using tools such as: 'webpack-bundle-analyzer'.",
```

Comment:
Nit: can you remove the `:` too? I don't think it makes sense anymore.

Reply:
wow that was fast, sure :)